openmetadata-ui/src/main/resources/ui/src/components/containers/PageLayoutV1.tsx

```diff
@@ -24,7 +24,7 @@ interface PageLayoutProp extends HTMLAttributes<HTMLDivElement> {
 export const pageContainerStyles = {
   height: '100%',
   padding: '1rem 0.5rem',
-
+  margin: 0,
```

Comment:
@Ashish8689 , This component is used in many places and we are adding the margin as 0 so please verify that we did not check-in unwanted alignment issues.

Reply:
@Sachin-chaurasiya yes, it is verified.